theme6/source/stylesheets/site.css.scss

```diff
@@ -1,159 +1,183 @@
-          background-color: #FBFBFB;
-        }
-        footer .author p{
-          margin-top: 35px;
-          font-size: 12px
-        }
-    }
+footer {
+  bottom: 0;
+  display: inline;
+  width: 100%;
+}
+
+.information {
+  justify-content: center;
+}
+
+li {
+  display: inline;
+}
+
+.social-media {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+}
+
+.sm {
+  height: 35px;
+  width: 93px;
+  border: 2px solid #72808C;
+  border-radius: 3px;
+}
+
+#fb {
+  background: url("../images/fb.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  margin-right: 23px;
+}
+
+#twt {
+  background: url("../images/twt.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+}
+
+#g {
+  background: url("../images/g+.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  margin-left: 23px;
+}
+
+.newsletter {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+  text-align: center;
+}
+
+footer .social-media-box {
+  align-content: center;
+  width: 350px;
+}
+
+footer .social-media-box button:hover {
+  cursor: pointer;
+}
+
+footer .social-media-container {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  width: 66%;
+}
+
+footer .social-media-container .logo {
+  margin-left: 96px;
+  margin-right: 96px;
+}
+
+footer .social-media-box p {
+  text-align: center;
+}
+
+footer .news-container {
+  width: 33%;
+  display: inline-flex;
+}
+
+footer .newsletter input {
+  height: 35px;
+  width: 300px;
+  justify-content: center;
+}
+
+footer .information ul li a {
+  height: 22px;
+  width: 192px;
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+  text-decoration: none;
+  align-items: center;
+  margin-right: 50PX;
+}
+
+footer .information ul li {
+  display: inline-block;
+}
+
+footer .information ul {
+  margin-top: 2%;
+  text-align: center;
+}
+
+footer .author {
+  display: block;
+  text-align: center;
+}
+
+footer .author p {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+@media screen and (max-width: 320px) {
+  .information {
+    display: none;
+  }
+
+  .logo {
+    display: none;
+  }
+
+  #twt {
+    margin-left: -20px;
+    margin-right: -20px;
+  }
+
+  .social-media-container {
+    display: flex;
+    margin-left: 50px;
+  }
+
+  .social-media-box {
+    align-content: center;
+  }
+
+  .social-media-box p {
+    font-size: 14px;
+  }
+
+  .social-media-icon-group {
+    width: 95%;
+    display: inline-flex;
+    justify-content: space-between;
+    align-content: center;
+  }
+
+  .news-container {
+    margin-top: 5px;
+    display: block;
+    align-content: center;
+    width: 100%;
+  }
+
+  .news-container p {
+    font-size: 15px;
+  }
+
+  .news-container input {
+    height: 40px;
```

Comment:
Properties should be ordered background-color, border, border-radius, height, width